changelog.md

```diff
@@ -25,13 +26,14 @@
 * Fixed admin pip requirements updater (#4955).
 * Added an environmental variable to control Docker build verbosity (#4943, #4974).
 * Increased libvirt prod VM memory to 1024MB (#4918).
+* Fixed CI LGTM alerts (#5019, #5026)`
```

Comment:
ah we didn't backport these LGTM alert PRs merged since we branched `release/1.2.0` off

Reply:
Argh, my bad. Updated and retagged.